src/AppInstallerCLITests/UserSettings.cpp

```diff
@@ -271,3 +271,44 @@ TEST_CASE("SettingAutoUpdateIntervalInMinutes", "[settings]")
         REQUIRE(userSettingTest.GetWarnings().size() == 1);
     }
 }
+
+// Test one experimental feature in usersettingstest context because there's no good way to test ExperimentalFeature
```

Comment:
>ExperimentalFeature [](start = 97, length = 19)

I think that you could have tests, but you would either need your Reload function back, or a test hook for features that allows you to inject your own settings object to be the one it uses.

Reply:
I kind of gave up after seeing all the amounts of changes i was doing plus me not wanting to mess with the constness of UserSettings::Instance

---
In reply to: [447942903](https://github.com/microsoft/winget-cli/pull/460#discussion_r447942903) [](ancestors = 447942903)